internal/app/api/v1/server.go

```diff
@@ -163,6 +163,7 @@ func (s *TestkubeAPI) Init(server server.HTTPServer) {
 	testWorkflows.Post("/:id/executions/:executionID/pause", s.PauseTestWorkflowExecutionHandler())
 	testWorkflows.Post("/:id/executions/:executionID/resume", s.ResumeTestWorkflowExecutionHandler())
 	testWorkflows.Get("/:id/executions/:executionID/logs", s.GetTestWorkflowExecutionLogsHandler())
+	testWorkflows.Post("/:id/executions/:executionID/rerun", s.ReRunTestWorkflowExecutionHandler())
```

Comment:
Take in account that this implementation is for OSS only, as (1) this API is basically deprecated (except when using OSS), (2) the whole scheduling is done on the Control Plane. For the Control Plane you need to prepare actual independent implementation.

Reply:
yes this is for oss